nxrm-ha-helm/templates/nexus-admin-secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.secret.nexusAdminSecret.create (not .Values.aws.secretmanager.enabled) (not .Values.azure.keyvault.enabled) }}
```

Comment:
`.Values.secret.nexusAdminSecret.create` change to `.Values.secret.nexusAdminSecret.enabled`

Reply:
I actually changed this to create because it was a little bit confusing using enabled. Basically we'll always have a secret object created for storing initial admin password. The question is whether it is auto created via the SecretManager or whether we create it manually using this file. That's why I thought changing this to `create` is clearer in this case.

Reply:
On second thought reverted to `enabled` for consistency with others : [revert variable name change in](https://github.com/sonatype/nxrm3-ha-repository/pull/21/commits/77d1c117c7e4b70e71d37725b4dbd3f6db195b95)